pkg/local_object_storage/engine/error_test.go

```diff
@@ -197,14 +197,13 @@ func checkShardState(t *testing.T, e *StorageEngine, id *shard.ID, errCount uint
 	require.Equal(t, errCount, sh.errorCount.Load())
 }
 
-// corruptSubDir makes random directory in blobstor FSTree unreadable.
+// corruptSubDir makes all directories in blobstor unreadable.
```

Comment:
It can affect tests a bit, can we just filter the blobovnicza dir in the `if`?

Reply:
Yes, but currently we need to hard-code it since dir organization is a private logic of `BlobStor`. I'll leave an issue.

Reply:
Done.